src/monetization.js

```diff
@@ -169,6 +170,40 @@ export class Monetization extends EventEmitter {
         return data;
     }
 
+    /**
+     * Checks if the user has access to a set of products or features.
+     * @param {array} pids - which products/features to check
+     * @param {number} userId - id of currently logged in user
+     * @throws {SDKError} - If a network call fails in any way (this will happen if, say, the user
+     * is not logged in)
+     * @returns {Object|null} The data object returned from Schibsted account (or `null` if the user
+     * doesn't have access to any of the given products/features)
+     */
+    async hasAccess(pids, userId) {
+        if (!this._sessionService) {
+            throw new SDKError("hasAccess can only be called if 'sessionDomain' is configured");
+        } else if (!userId) {
+            throw new SDKError("'userId' must be specified");
+        } else if (!Array.isArray(pids)) {
+            throw new SDKError("'pids' must be an array");
+        }
+
+        const sortedIds = pids.sort();
+        const cacheKey = `prd_${sortedIds}_${userId}`;
```

Comment:
since `sortedIds` is a array by default it will use `,` to join items. If you want use other separator you can use `.join()`

Reply:
I think comma-separated is fine for the cache key.